vn.hoabinh.quan.CoBang.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: vn.hoabinh.quan.CoBang
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: cobang
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
```

Comment:
What does it need host filesystem access for?

Reply:
There is a feature that let user to add an image file from filesystem. Maybe it should be:

```
--filesystem=host:ro
```
?

I don't know what is difference between `filesystem=host` and `filesystem=home`.

Reply:
If all you need is to open a file, you can use Gtk.FileChooserNative which will use the document portal and you won't need access to the whole host/home thing. You will just have access to whatever files that were opened by the user using your application. 
In the meantime, I think home:ro makes more sene. The permissions can be updated by the user if they want to give it access to mounted devices or whatever using Flatseal or flatpak override.